plugins/modules/redfish_config.py

```diff
@@ -272,6 +278,20 @@
       baseuri: "{{ baseuri }}"
       username: "{{ username }}"
       password: "{{ password }}"
+
+  - name: Create Volume
+    community.general.redfish_config:
+      category: Systems
+      command: CreateVolume
```

Comment:
How does the user specify which storage subsystem will contain the new volume?

Reply:
Shall we take subsystem ID from the user as input?
e.g. StorageSubsystemID: "DE00B000"

Reply:
That would be a good addition.

Reply:
Changed as discussed.

Reply:
Resolving this conversation. Thanks!